sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -210,6 +212,19 @@ object DataSourceV2Strategy extends Strategy with PredicateHelper {
     case AlterNamespaceSetProperties(catalog, namespace, properties) =>
       AlterNamespaceSetPropertiesExec(catalog, namespace, properties) :: Nil
 
+    case CommentOnNamespace(ResolvedNamespace(catalog, namespace), comment) =>
```

Comment:
After some more thoughts, I think it's true that empty string comment is the same as no comment w.r.t. DESCRIBE command. But it's different to catalog developers who can access the properties directly. I think it's better to remove the comment property if the comment string is empty.

Reply:
This makes things a bit complicated and inconsistent. With our default implementation now in session catalog, the reserved props are unremovable https://github.com/apache/spark/blob/a0057131690b65b73aa6310500039d3a92466324/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala#L230-L235.
If we take null as empty string and update, then the logic between catalogs can be consistent  which also protected by the underlying  command `AlterNamespaceSetPropertiesExec`
https://github.com/apache/spark/blob/c32e2286898fe8660c7deed9303f1c5c15b60757/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/AlterNamespaceSetPropertiesExec.scala#L32-L36

Reply:
OK let's keep it as it is and revisit it later. I think we should only disallow end-users to explicitly remove reserved properties via `ALTER TABLE UNSET TBLPROPERTIES`.

Reply:
yes, I agree with that, we also may need to introduce `Alter Namespace Unset Properties` command too